### DISCUSSION TITLE
How to resolve (Error: Couldn't retrieve archiver key for gID __ gToken __)

### DISCUSSION THREAD

**Discussion by wpefover11** [Difegue/LANraragi#1104] [Q&A] [answered]
I am encountering an issue with EH when attempting to use the Tsukihi extension to download a tab. There was a change in behavior in the last week or so. When attempting to initiate a download, it returns the message: 
(Error: Couldn't retrieve archiver key for gID __ gToken __)

I've tried to diagnose from search existing discussions, but I haven't found anything related to archiver keys. Is there anything on the LLR client that I am not using correctly? Or is it something on the EH end that may be causing this?

I've checked the following:

I was already on the latest version, but I checked the update statuses and I confirmed I have the following versions.
- Version 0.9.21
- WSL 2
- Tsukihi 2.0

I'm logged into EH and able to initiate downloads manually without issue. There is no issue with credits or GP on my login.

The login plugin for EH has the cookie information copied directly for the following parameters
- ipb_member_id cookie
- ipb_pass_hash cookie 
- star cookie

Both LRR and Tsukihi are functioning as normal with Chaika.

I've attempted to reset LLR settings and repair the installation for LRR as well. This has not fixed the issue.

**Comment by Difegue** [comment, marked as answer]
See https://github.com/Difegue/LANraragi/issues/1101.

**Reply by wpefover11** [reply]
Ok, thank you for the fast response.
